dist/routable-component.js

```diff
@@ -381,37 +381,37 @@ var routableComponent =
 	var RoutableComponentRoutes = function () {
 	  /**
 	   * コンストラクタ
-	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routing_classes ルート定義クラス(の配列)
+	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routingClasses ルート定義クラス(の配列)
 	   */
 	
 	  function RoutableComponentRoutes() {
-	    var routing_classes = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
+	    var routingClasses = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
 	    (0, _classCallCheck3.default)(this, RoutableComponentRoutes);
 	
 	    this._routes = [];
-	    this.include_route(routing_classes);
+	    this.includeRoute(routingClasses);
 	  }
 	
 	  /**
 	   * ルートを設定する
-	   * @param {Route|Route[]} routing_classes ルート定義クラス(の配列)
+	   * @param {Route|Route[]} routingClasses ルート定義クラス(の配列)
 	   * @return {void}
 	   */
 	
 	
 	  (0, _createClass3.default)(RoutableComponentRoutes, [{
-	    key: 'include_route',
-	    value: function include_route(routing_classes) {
-	      var _routing_classes = routing_classes instanceof Array ? routing_classes : [routing_classes];
+	    key: 'includeRoute',
+	    value: function includeRoute(routingClasses) {
+	      var _routingClasses = routingClasses instanceof Array ? routingClasses : [routingClasses];
 	      var _iteratorNormalCompletion5 = true;
 	      var _didIteratorError5 = false;
 	      var _iteratorError5 = undefined;
 	
 	      try {
-	        for (var _iterator5 = (0, _getIterator3.default)(_routing_classes), _step5; !(_iteratorNormalCompletion5 = (_step5 = _iterator5.next()).done); _iteratorNormalCompletion5 = true) {
-	          var route_class = _step5.value;
+	        for (var _iterator5 = (0, _getIterator3.default)(_routingClasses), _step5; !(_iteratorNormalCompletion5 = (_step5 = _iterator5.next()).done); _iteratorNormalCompletion5 = true) {
+	          var routeClass = _step5.value;
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)
* Warning - All 'var' declarations must be at the top of the function scope. (vars-on-top)
* Warning - Unexpected var

<!-- Comment by [SideCI](https://sideci.com) -->